src/main/java/core/basesyntax/strategy/SupplyHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class SupplyHandler implements OperationHandler {
+    private static final String SUPPLY = "s";
+
+    @Override
+    public List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> listOfAll,
```

Comment:
The method signature for 'getFruitFinalQuantity' is a bit confusing. It's not clear what 'listOfAll' and 'listOfFruits' represent just from their names. Consider renaming these parameters to more descriptive names that indicate their roles.